tardis/io/logger/logger.py

```diff
@@ -136,31 +67,146 @@ def logging_state(log_level, tardis_config, specific_log_level):
                 logger.removeFilter(filter)
 
     if specific_log_level:
-        filter_log = FilterLog(LOGGING_LEVELS[logging_level])
+        filter_log = FilterLog([LOGGING_LEVELS[logging_level], logging.INFO, logging.DEBUG])
         for logger in tardis_loggers:
             logger.addFilter(filter_log)
     else:
         for filter in logger.filters:
             for logger in tardis_loggers:
                 logger.removeFilter(filter)
 
+log_outputs = {
+    "WARNING/ERROR": Output(layout=Layout(height='300px', overflow_y='auto')),
+    "INFO": Output(layout=Layout(height='300px', overflow_y='auto')),
+    "DEBUG": Output(layout=Layout(height='300px', overflow_y='auto')),
+    "ALL": Output(layout=Layout(height='300px', overflow_y='auto'))
```

Comment:
you can define a function that returns a Output widget and call for each of the keys, instead of repeating code.